.contrib/Parser/lib/Strings/locales/Default Locale/INCOMPLETE_ICON.lua

```diff
@@ -4,6 +4,16 @@ createLocalizationString({
 	icon = [[~_.asset("incomplete")]],
 	export = true,
 	text = {
-		en = "",
+		en = "Incomplete",
```

Comment:
This should also stay en = "",

This locale is specifically intended to only be the icon, so no localization is required. This is just how we can get parser to build a locale constant for it.